src/DefaultVerticalNodeGeometry.cpp

```diff
@@ -32,7 +32,10 @@ void DefaultVerticalNodeGeometry::recomputeSize(NodeId const nodeId) const
 {
     unsigned int height = _portSpasing; // maxHorizontalPortsExtent(nodeId);
 
-    if (auto w = _graphModel.nodeData<QWidget *>(nodeId, NodeRole::Widget)) {
+    bool isEmbeded = _graphModel.nodeData(nodeId, NodeRole::WidgetEmbeddable).value<bool>();
+    auto w = _graphModel.nodeData<QWidget *>(nodeId, NodeRole::Widget);
+
+    if (isEmbeded && w) {
```

Comment:
`isEmbeded` -> `isEmbedded`

Please rename all instances of this variable to the correct spelling